declarations_site/catalog/templates/_youcontrol.jinja

```diff
@@ -5,7 +5,7 @@
     </svg>
 {% endmacro %}
 
-{% macro yc_check_person(option, class_name = '', size = 'sm') %}
+{% macro yc_check_person(option, class_name = '', size = 'sm', name = '', page = '') %}
     {% if size == 'sm' %}
         {% if option == 'a' %}
         <a href="#" class="{{ class_name }} youcontrol__check-person youcontrol__check-person--option-a youcontrol__check-person--sm" title="{{ _("Досьє від Youcontrol") }}">
```

Comment:
Links here and below (except for the option B) are left empty (#) is that a bug or it was left as is intentionally?

Reply:
It's OK, coz we are testing only option B. But we keep another option which may have different links for the future